src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/account_filter.py

```diff
@@ -14,13 +14,17 @@
 
 def create_account_filter(client, account_name, resource_group_name, filter_name, start_timestamp=None,
                           end_timestamp=None, presentation_window_duration=None, live_backoff_duration=None,
-                          timescale=None, force_end_timestamp=False, bitrate=None, tracks=None):
+                          timescale=None, force_end_timestamp=False, bitrate=None, firstQuality=None, tracks=None):
```

Comment:
This should be `first_quality` and exposed as `--first-quality`.

Reply:
Thanks. I'll make that change. Originally, left some of the calls to filter with the bitrate parameter instead of updating them to use first-quality. I'll update all of them except one so that I see the deprecation message on one test rather than the 7 or 8 tests that use that bitrate.